pycalphad/core/minimizer.pyx

```diff
@@ -617,9 +622,10 @@ cpdef take_step(SystemSpecification spec, SystemState state, double step_size):
         state.chemical_potentials[comp_idx] = spec.initial_chemical_potentials[comp_idx]
 
     # Update phase internal degrees of freedom
-    for idx, compset in enumerate(state.compsets):
+    for idx in range(len(state.compsets)):
         # TODO: Use better dof storage
         x = state.dof[idx]
+        compset = state.compsets[idx]
```

Comment:
I don't think it's necessary to assign `compset` here.

Reply:
Removed